sites/bizbash.com/server/routes/website-section.js

```diff
@@ -15,6 +15,10 @@ module.exports = (app) => {
     template: venueDirectory,
     queryFragment,
   }));
+  app.get('/:alias(live-events)', withWebsiteSection({
```

Comment:
@jwade1327 The route the /events page is hitting is not the one below.  it is actually https://github.com/parameter1/bizbash-media-websites/blob/master/sites/bizbash.com/server/routes/published-content.js#L67. 

This route loads a specific page that loads items, first from /virtual-event & then items from /live-events.  If it is easier to just add :alias(events||live-events||virtual-events) to that route and add logic to that template to only display <@section> based off of the given alias.  that might be an option.